.clang-format

```diff
@@ -0,0 +1,116 @@
+---
+BasedOnStyle:  WebKit
+# AccessModifierOffset: -4
+# AlignAfterOpenBracket: DontAlign
+# AlignConsecutiveAssignments: false
```

Comment:
do we need to keep all these commented out statements?

Reply:
it helps to know the webkit original parameters ...